core/src/test/java/io/parsingdata/metal/expression/value/ExpandTest.java

```diff
@@ -99,7 +99,7 @@ public void expandList() {
         }
     }
 
-    private static ParseValue createParseValue(String name, int value) {
+    public static ParseValue createParseValue(String name, int value) {
```

Comment:
Can this perhaps be moved, as it is used across multiple classes, and not really specifically belongs to this class?